esbuild.js

```diff
@@ -1,10 +1,26 @@
 const esbuild = require('esbuild');
+const ignorePlugin = require('esbuild-plugin-ignore');
+const textReplacePlugin = require('esbuild-plugin-text-replace');
 
 esbuild
   .build({
     bundle: true,
     outfile: 'dist/worker.js',
     entryPoints: ['src/index.ts'],
+    plugins: [
+      ignorePlugin([
+        {
+          resourceRegExp: /\.wasm$/,
+        },
+      ]),
+      textReplacePlugin({
+        include: /brotli_wasm(?:_bg?)\.js$/,
+        pattern: [
+          ["import * as wasm from './brotli_wasm_bg.wasm';", ''],
+          ['wasm.', 'globalThis.__BROTLI_WASM_INSTANCE.'],
```

Comment:
Ok. I wonder if this would be easier if we switched to module workers, which support importing WASM without anything special. https://blog.cloudflare.com/workers-javascript-modules/